### PR TITLE
Add the ability to manage Auto-Close on issues

### PR DESCRIPTION
**Pull request by RickWinter**
This PR is providing a way to skip the Enforce Max Life of Issues

### CODE REVIEW

File: tools/github/data/common-labels.csv
Comment (JimSuplizio) [inline tools/github/data/common-labels.csv:231]: @RickWinter, I'm going to defer to @jsquire and @ronniegeraghty on the color of this label. I'm sure one or both will want a different color.

**Review by JimSuplizio — APPROVED** [review]
As per our conversation, I'm fine with the changes. You added the label constant and then added it to the query's exclude list which is exactly what needed to be done, thank you.

@jsquire or @ronniegeraghty, one of you is probably going to have to run the sync script so the auto-close-exempt label gets added to all of the repositories after this PR gets merged.

**Comment by jsquire** [comment]
@ronniegeraghty:  Once you tell me we're good, I'll do a run to push that label out to the managed repos.

File: tools/github/data/common-labels.csv
Comment (ronniegeraghty) [reply tools/github/data/common-labels.csv:231]: I think the color is fine. It's very grey and dull, and I don't really want to bring any attention to this label. I want it to be a tool only the SDK team uses so an unassuming color works here.

**Review by ronniegeraghty — APPROVED** [review]

**Comment by ronniegeraghty** [comment]
@jsquire, @RickWinter, @JimSuplizio 
All looks good to me.

File: tools/github/data/common-labels.csv
Comment (jsquire) [reply tools/github/data/common-labels.csv:231]: I'm going to ask that we adjust to match the other greyed out labels, just for consistency.  That's the form that I've pushed out to the other repos.

Comment (jsquire) [inline tools/github/data/common-labels.csv:231]:
```suggestion
auto-close-exempt,Prevents the auto-close from closing based on max lifetime,ededed
```

**Comment by weshaggard** [comment]
I believe this is for issue https://github.com/Azure/azure-sdk-tools/issues/8559

**Pull request merged**
